ethstorage/miner/worker.go

```diff
@@ -312,7 +321,9 @@ func (w *worker) resultLoop() {
 				*result,
 				w.config,
 			)
+			totalSubmitted++
 			if err != nil {
+				errorCache = append(errorCache, miningError{result.startShardId, result.blockNumber, err})
```

Comment:
this only contains L1API submit result error, may need to include other errors during mining/generating results.